src/index.html

```diff
@@ -5,8 +5,310 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
+  <header class="header">
+    <div class="header__logo">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo.png">
+      </a>
+    </div>
+    <nav class="nav">
```

Comment:
Между тегами одного уровня с достаточно объемным содержанием весьма желательны доп. пустые строки. Просмотри здесь и далее.
```suggestion
    </div>
    
    <nav class="nav">
```